lib/puppet/type/cisco_aci_vzsubj.rb

```diff
@@ -66,30 +75,32 @@ def self.title_patterns
   # ---------------------------------------------------------------
 
   newproperty(:cons_match_t) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzSubj.html#consMatchT'
+    desc 'The subject match criteria across consumers.'
   end
 
   newproperty(:descr) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzSubj.html#descr'
+    desc 'Specifies the description of a policy component.'
   end
 
   newproperty(:name_alias) do
-    desc 'https://pubhub.devnetcloud.com/media/apic-mim-ref-311/docs/MO-vzSubj.html#nameAlias'
+    desc ''
```

Comment:
IMO, we should fill in something here, even if it's simply `Specifies an alias for the managed object`.  Empty strings just look incomplete.